docs/source/howto/archive_profile.md

```diff
@@ -20,69 +20,50 @@ This tutorial can be downloaded and run as a Jupyter Notebook: {nb-download}`arc
-but is used within a context manager, that insures that the storage is properly closed when the context is exited.
-With this, we can load our archive as a profile:
-
-```{code-cell} ipython3
-with profile_context(archive_profile):
-    print(manage.get_manager().get_profile())
-```
 
-To directly access the storage backend, and view information about it, we can use:
+The easiest way to inspect the contents of an archive is to create a profile that "mounts" the archive as its data storage:
 
 ```{code-cell} ipython3
-import json
-with profile_context(archive_profile):
-    storage = manage.get_manager().get_profile_storage()
-    print(storage)
-    print(json.dumps(storage.get_info(), indent=2))
+!verdi profile setup core.sqlite_zip -n --profile archive --filepath include/process.aiida
 ```
 
-This is directly equivalent to the command-line call:
+You can now inspect the contents of the `process.aiida` archive by using the `archive` profile in the same way you would a standard AiiDA profile.
+For example, you can start an interactive shell using `verdi -p archive shell` or if you are already in a notebook simply load the profile:
 
 ```{code-cell} ipython3
-!verdi archive info include/process.aiida
+from aiida import load_profile
+load_profile('archive', allow_switch=True)
 ```
 
-Note, that once the context manager is exited, the storage is closed, and will except on further calls.
-
-```{code-cell} ipython3
-print(storage)
+```{warning}
+A profile using the `core.sqlite_zip` storage is read-only.
+It is therefore possible to query data, but trying to modify existing data or store new data will raise an exception.
 ```
 
-As per a standard profile, we can now use the {py:class}`~aiida.orm.QueryBuilder`, to [find and query for data](how-to:query):
+Just as with a normal profile, we can now use the {py:class}`~aiida.orm.QueryBuilder`, to [find and query for data](how-to:query):
 
 ```{code-cell} ipython3
-with profile_context(archive_profile):
-    process = orm.QueryBuilder().append(orm.ProcessNode).first(flat=True)
-    print(process)
+from aiida import orm
+process = orm.QueryBuilder().append(orm.ProcessNode).first(flat=True)
+print(process)
 ```
 
 and also use {py:class}`~aiida.tools.visualization.graph.Graph`, to [visualize data provenance](how-to:data:visualise-provenance):
 
 ```{code-cell} ipython3
+from aiida import orm
 from aiida.tools.visualization import Graph
-
-with profile_context(archive_profile):
-    process = orm.QueryBuilder().append(orm.ProcessNode).first(flat=True)
-    graph = Graph(graph_attr={"size": "8!,8!", "rankdir": "LR"})
-    graph.add_incoming(process, annotate_links="both")
-    graph.add_outgoing(process, annotate_links="both")
-
+process = orm.QueryBuilder().append(orm.ProcessNode).first(flat=True)
+graph = Graph(graph_attr={'size': '8!,8!', 'rankdir': 'LR'})
```

Comment:
```suggestion
graph = Graph(graph_attr={'rankdir': 'LR'})
```

Looks set the size will cause the render problem in doc. Not only here but for other graphs in https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/visualising_graphs.html, we likely want to fix all of them.

Reply:
Done.